docs/source/customization.rst

```diff
@@ -145,3 +148,71 @@ look for possible overrides using the name of the model itself, but also using
 all the parent models following the full inheritance chain. So if all your models
 inherit from ``MyAbstractModel``, you can for example create an override table
 for all your models by creating a ``myproject.tables.MyAbstractModelTable``.
+
+
+Extending autocomplete results
```

Comment:
This duplicates in part the part from line 115-120, so it might make sense to delete those lines

Reply:
yes, but it contains actual example code....trying to move it in the upper section

Reply:
Yes, I meant removing the older version or merging it together